controller/controller-common/src/main/java/com/pinterest/rocksplicator/controller/TaskQueue.java

```diff
@@ -143,9 +143,10 @@ default boolean removeCluster(final String cluster) {
 
   /**
    * Remove finished tasks.
+   * @param secondsAgo tasks finished seconds ago
    * @return the number of finished tasks which are removed by this call.
    */
-  default int removeFinishedTasks() {
+  default int removeFinishedTasks(final int secondsAgo) {
```

Comment:
i think our default time unit is all seconds if you look at other api and design doc.